mobile_codes/json/mnc_operators.json

```diff
@@ -1,1344 +1,1933 @@
 [
-    ["001", "01", "TEST", "Test Network"],
-    ["202", "01", "Cosmote", "Cosmote"],
-    ["202", "02", "Cosmote", "Cosmote"],
-    ["202", "03", "OTE", "OTE"],
-    ["202", "04", "EDISY", "EDISY"],
-    ["202", "05", "Vodafone - Panafon", "Vodafone - Panafon"],
-    ["202", "06", "COSMOLINE", "COSMOLINE"],
-    ["202", "07", "AMD TELECOM", "AMD TELECOM"],
-    ["202", "09", "WIND", "WIND"],
-    ["202", "10", "WIND", "WIND"],
-    ["204", "01", "RadioAccess Network Services B.V.", "RadioAccess Network Services B.V."],
-    ["204", "02", "Tele2 Nederland B.V.", "Tele2 Nederland B.V."],
-    ["204", "03", "Voiceworks B.V.", "Voiceworks B.V."],
-    ["204", "04", "Vodafone Libertel B.V.", "Vodafone Libertel B.V."],
-    ["204", "05", "Elephant Talk Communications Premium \nRate Services", "Elephant Talk Communications Premium \nRate Services"],
-    ["204", "06", "Mundio Mobile (Netherlands) Ltd", "Mundio Mobile (Netherlands) Ltd"],
-    ["204", "07", "Teleena holding B.V.", "Teleena holding B.V."],
-    ["204", "08", "KPN Mobile The Netherlands B.V.", "KPN Mobile The Netherlands B.V."],
-    ["204", "09", "Lycamobile Netherlands Limited", "Lycamobile Netherlands Limited"],
-    ["204", "10", "KPN B.V.", "KPN B.V."],
+    ["001", "001", "TEST", "Test network"],
+    ["001", "01", "TEST", "Test network"],
+    ["202", "01", "Cosmote", "COSMOTE - Mobile Telecommunications S.A."],
+    ["202", "02", "Cosmote", "COSMOTE - Mobile Telecommunications S.A."],
```

Comment:
Previously, the brand and operator were generally the same string. These are now different, parsed from the Wikipedia source.

Reply:
Yes in many cases they was the same, because the main original source itu.json, (derived from the itu publication) does not have a distinct column for brand/operator, and it seems the original project just copied the same information in both fields. But used the wikipedia data as an updated over that initial import. The same is true also for the added additional data source mcc-mnc-table.json.